agent/containers/images/visualizers/prom-graf-visualizer/prom_ds.py

```diff
@@ -0,0 +1,65 @@
+import json
```

Comment:
Can we merge this `prom_ds.py` with the other one and use switches or environment variables to control the desired behavior?